controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -206,7 +206,8 @@ http {
         {{ if eq $upstream.SessionAffinity.AffinityType "cookie" }}
         sticky hash={{$upstream.SessionAffinity.CookieSessionAffinity.Hash}} name={{$upstream.SessionAffinity.CookieSessionAffinity.Name}}  httponly;
         {{ else }}
-        least_conn;
+        # Load balance algorithm; empty for round robin, which is the default
+        {{ $cfg.LoadBalanceAlgorithm }}
```

Comment:
yeah it does, but https://github.com/kubernetes/ingress/pull/673#pullrequestreview-35464498 seems like a good suggestion, yeah?

Another option is to use conditionals in the template and only set the load balancing policy if it's not `round_robin`.  E.g., similar to what the session affinity check above does.